protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -73,6 +73,31 @@ definitions:
       emitted_at:
         description: "when the data was emitted from the source. epoch in millisecond."
         type: integer
+      meta:
+        description: Information about this record added mid-sync
+        "$ref": "#/definitions/AirbyteRecordMessageMeta"
+  AirbyteRecordMessageMeta:
+    type: object
+    additionalProperties: true
+    properties:
+      errors:
+        description: Key-Value pairs of problems for properties of this record
+        type: array
+        items:
+          "$ref": "#/definitions/AirbyteRecordMessageMetaErrors"
+  AirbyteRecordMessageMetaErrors:
+    type: object
+    additionalProperties: true
+    required:
+      - key
+      - message
+    properties:
+      key:
+        type: string
+      message:
+        type: string
+      origin:
```

Comment:
what's this? Is it an `enum` of source, platform, destination?

Reply:
good call! added in f19499949143425713d497124a01b0cca7353d69

Reply:
... we didn't have that enum in the protocol anywhere yet 🤷?